pages/linux/pactl.md

```diff
@@ -0,0 +1,23 @@
+# pactl
+
+> Control a running PulseAudio sound server.
+
+- List all sinks (or other types - sinks are outputs and sink-inputs are active audio streams):
+
+`pactl list {{sinks}} short`
```

Comment:
How many types can there be? If only a few, I'd suggest listing them in the token using the pipe syntax, e.g. `{{sinks|sink-inputs}}`.

Reply:
Eight types I think. cards, clients, modules, samples, sink-inputs, sinks, source-outputs, sources.

Though for someone reading the tldr I think sinks, sink-inputs, sources and source-outputs would be the most interesting.

I'm not entirely sure what you can do with clients, modules and samples to be honest, they're probably useful for people who know what they're doing but I don't think they do much for an average user.

Reply:
I'm worrying that if we present a shorter list like `{{sinks|sink-inputs|sources|source-outputs}}`, it might induce people in error, as if we're suggesting no other values are possible (and it's not that compact either). I guess it makes more sense to keep this example as it is, but let me know if you think otherwise.